tests/default/data/doctrine-dbal-union-result.php

```diff
@@ -26,14 +26,4 @@ public function doFoo(Connection $conn)
             assertType('array{adaid: int<-32768, 32767>}|array{email: string}|false', $fetch);
         }
     }
-
-    public function doBar(Connection $conn)
```

Comment:
looks like you removed code from tests which previously were run under doctrine/dbal 2.x and 3.x, but re-added it in a file which only is run on 3.x?